.github/workflows/container_image.yml

```diff
@@ -0,0 +1,45 @@
+name: Create container image
+on: [push]
+
+jobs:
+  create_container_image:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ghcr.io/${{ github.repository }}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=tag
+            type=ref,event=pr
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        id: buildx
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build container image
+        uses: docker/build-push-action@v3
+        with:
+          # We don't use git context because that doesn't process .dockerignore
+          # https://github.com/docker/cli/issues/2827
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          secrets: |
+            "github_token=${{ secrets.RDO_REPO_RO }}"
```

Comment:
Should this not just use the github_token used above?

Reply:
After consulting @dekkers, decided to try without it. Seems to work.